src/measure_theory/group/action.lean

```diff
@@ -0,0 +1,114 @@
+/-
+Copyright (c) 2021 Yury G. Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury G. Kudryashov
+-/
+import measure_theory.group.measurable_equiv
+import measure_theory.measure.regular
+import dynamics.ergodic.measure_preserving
+import dynamics.minimal
+
+/-!
+# Measures invariant under group actions
+
+Measure `μ : measure α` is said to be *invariant* under action of a group `G` if scalar
+multiplication by `c : G` is a measure preserving map for all `c`. In this file we define a
+typeclass for measures invariant under action of an (additive or multiplicative) group and prove
+some basic properties of such measures.
+-/
+
+open_locale ennreal nnreal pointwise topological_space
+open measure_theory measure_theory.measure set function
+
+namespace measure_theory
+
+variables {G M α : Type*} [measurable_space α]
+
+/-- Measure `μ : measure α` is invariant under an additive action of `M` on `α` if for any
+measurable set `s : set α` and `c : M`, the measure of its preimage under `λ x, c +ᵥ x` is equal to
+the measure of `s`. -/
+class vadd_invariant_measure (M α : Type*) [has_vadd M α] [measurable_space α]
+  (μ : measure α) : Prop :=
```

Comment:
In most of this file, you have `[measurable_space α] (μ : measure α)`. In this kind of situation, can you turn the first argument from typeclass to implicit?

Reply:
Done.